tests/test_note.py

```diff
@@ -26,7 +26,6 @@
         ("C", 0),
         ("C#", 1),
         ("Db", 1),
-        ("C###", 3),
```

Comment:
We now use the same regex on different execution paths, so triple accidentals will no longer parse.

Reply:
That's fine, working for arbitrary combos of `#b=` in `pitch_class_value` was only for fun anyway, not very useful. Maybe I'll bring it back as an option later.